sharding/indexHashedNodesCoordinator.go

```diff
@@ -805,6 +840,20 @@ func (ihgs *indexHashedNodesCoordinator) computeShardForSelfPublicKey(nodesConfi
 		}
 	}
 
+	for shard, validators := range nodesConfig.leavingMap {
+		for _, v := range validators {
```

Comment:
a function can be made like searchInMap. or use searchInMap function here as well.duplicated code

Reply:
fixed